docs/development.rst

```diff
@@ -135,6 +135,15 @@ There are many ways to filter tests when running CTest, or run the tests in
 parallel. More information about CTest can be found
 `here <http://www.cmake.org/cmake/help/v3.0/manual/ctest.1.html>`_.
 
+If you run into errors on any of the packaging tests, two possible fixes are
+
+1) run ``make`` inside your ``girder-build`` directory, which will create a special
+virtualenv needed to build the packages.
+
+2) delete any of the files generated by the packaging tests, which will be in your
+source dir ``girder`` and could include ``girder-1.2.4.tar.gz``, ``girder-web-1.2.4.tar.gz``,
+and ``girder-plugins-1.2.4.tar.gz``.
```

Comment:
We should probably just list these as girder-<version>.tar.gz instead of hard-coding the version, so we don't have to worry about keeping these docs up to date with each release.